patch-package
--- a/node_modules/eslint-config-cat/index.js
+++ b/node_modules/eslint-config-cat/index.js
@@ -1,11 +1,5 @@
 'use strict';
 
-const path = require('path');
-
-const {alias} = require(
-  path.resolve(process.cwd(), './.babelrc')
-);
-
 module.exports = {
   extends: [
     'plugin:flowtype/recommended',
@@ -23,7 +17,7 @@ module.exports = {
   ],
   settings: {
     'import/resolver': {
-      'babel-module': alias
+      'babel-module': {}
     },
     flowtype: {
       onlyFilesWithFlowAnnotation: true
